src/image/Filter.cc

```diff
@@ -35,8 +35,6 @@
 #include "lsst/daf/base/PropertySet.h"
 
 #include "lsst/afw/image/Filter.h"
-
-#include "lsst/afw/table/io/InputArchive.h"
```

Comment:
InputArchive is actually used here

Reply:
InputArchive is declared in Persistable.h and member of table::io::PersistableFactory